configure_user/scripts/add_user.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
```

Comment:
Add set -e to force the tf execution to fail if the script error out.

Reply:
done